source/cloud/azure/aks.md

```diff
@@ -23,7 +23,7 @@ $ az login
 Now we can launch a GPU enabled AKS cluster. First launch an AKS cluster.
 
 ```console
```

Comment:
As this is not a `console` style example (command with output) maybe we should tell the highlighter that it's `bash` instead.

```suggestion
```bash
```